From 39451592c92547ab419d43652014a9bff5be1283 Mon Sep 17 00:00:00 2001
From: Son Lam <son.lam.ym@renesas.com>
Date: Thu, 30 May 2019 13:27:17 +0700
Subject: [PATCH 182/215] clk: renesas: r8a774a1-cpg-mssr: add ivdp1c, vcplf,
 vdpb and 3dge clock

This commit add:
- VCP4 (ivdp1c, vcplf, vdpb) clock
- 3DGE clock

Signed-off-by: Son Lam <son.lam.ym@renesas.com>
---
 drivers/clk/renesas/r8a774a1-cpg-mssr.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/drivers/clk/renesas/r8a774a1-cpg-mssr.c b/drivers/clk/renesas/r8a774a1-cpg-mssr.c
index 86be10f..933be56 100644
--- a/drivers/clk/renesas/r8a774a1-cpg-mssr.c
+++ b/drivers/clk/renesas/r8a774a1-cpg-mssr.c
@@ -73,6 +73,7 @@ static const struct cpg_core_clk r8a774a1_core_clks[] __initconst = {
 	/* Core Clock Outputs */
 	DEF_BASE("z",		R8A774A1_CLK_Z,	 CLK_TYPE_GEN3_Z,  CLK_PLL0, 2),
 	DEF_BASE("z2",		R8A774A1_CLK_Z2, CLK_TYPE_GEN3_Z2, CLK_PLL2, 2),
+	DEF_BASE("zg",		R8A774A1_CLK_ZG, CLK_TYPE_GEN3_ZG, CLK_PLL4, 4),
 	DEF_FIXED("ztr",	R8A774A1_CLK_ZTR,   CLK_PLL1_DIV2,	6, 1),
 	DEF_FIXED("ztrd2",	R8A774A1_CLK_ZTRD2, CLK_PLL1_DIV2,	12, 1),
 	DEF_FIXED("zt",		R8A774A1_CLK_ZT,    CLK_PLL1_DIV2,	4, 1),
@@ -111,7 +112,11 @@ static const struct cpg_core_clk r8a774a1_core_clks[] __initconst = {
 };
 
 static const struct mssr_mod_clk r8a774a1_mod_clks[] __initconst = {
+	DEF_MOD("3dge",			112,	R8A774A1_CLK_ZG),
 	DEF_MOD("fdp1-0",		119,	R8A774A1_CLK_S0D1),
+	DEF_MOD("ivdp1c",		128,	R8A774A1_CLK_S0D2),
+	DEF_MOD("vcplf",		130,	R8A774A1_CLK_S0D2),
+	DEF_MOD("vdpb",			131,	R8A774A1_CLK_S0D2),
 	DEF_MOD("tmu4",			121,	R8A774A1_CLK_S0D6),
 	DEF_MOD("tmu3",			122,	R8A774A1_CLK_S3D2),
 	DEF_MOD("tmu2",			123,	R8A774A1_CLK_S3D2),
-- 
2.7.4

